packages/components/src/internal/schemas.ts

```diff
@@ -107,6 +107,13 @@ export const SAMPLE_SETS = {
     SAMPLES: SchemaQuery.create(SAMPLE_SET_SCHEMA, 'Samples'),
 };
 
+// SAMPLE MANAGEMENT
+const SAMPLE_MANAGEMENT_SCHEMA = 'sampleManagement';
+export const SAMPLE_MANAGEMENT = {
+    SCHEMA: SAMPLE_MANAGEMENT_SCHEMA,
+    SOURCE_SAMPLES: SchemaQuery.create(SAMPLE_MANAGEMENT_SCHEMA, "SourceSamples"),
```

Comment:
I assume this is the only query you need for this story, but there are likely others in the definition of this const in the SM module code (or maybe in workflow package). Thoughts on moving over the full `SAMPLE_MANAGEMENT` definition?